Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py

```diff
@@ -126,6 +126,7 @@ def http_request(url: str, method: str, headers: dict = None, body=None, params=
         json_res = json.loads(xml2json(result.text))
         return json_res
     except Exception:
+        demisto.debug('Failed to parse response to json.')
```

Comment:
relevant for all logs in an exception clause.
```suggestion
    except Exception as exc:
        demisto.error(f'Failed to parse response to json. Error: {exc}')
```